.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-5.3.0
\ No newline at end of file
+5.2.1
```

Comment:
Why the down grade?

Reply:
I am not sure, it's probably some kind of janky merge thing